src/wp-includes/default-constants.php

```diff
@@ -377,6 +377,13 @@ function wp_functionality_constants() {
 	if ( ! defined( 'WP_CRON_LOCK_TIMEOUT' ) ) {
 		define( 'WP_CRON_LOCK_TIMEOUT', MINUTE_IN_SECONDS );
 	}
+
+	/**
+	 * @since 6.0.0
+	 */
+	if ( ! defined( 'WP_OBJECT_CACHE_IGNORE_GLOBAL_GROUPS' ) ) {
+		define( 'WP_OBJECT_CACHE_IGNORE_GLOBAL_GROUPS', array() );
```

Comment:
As previously noted, assigning an array to a constant is not supported until PHP 7.0. Though a bump will come, currently WordPress also supports PHP 5.6. I'm not a fan of mixing types; however, in this case, a constant is the best approach when targeting configuration in the `wp-config.php` file. So an empty value can mean it's not set and don't use it. Instead of `array()` as the default, I'd suggest `false` as the flag to know that this constant is not configured for usage.